xlators/cluster/afr/src/afr-common.c

```diff
@@ -731,16 +731,13 @@ afr_copy_frame(call_frame_t *base)
 /* Check if an entry or inode could be undergoing a transaction. */
 static gf_boolean_t
 afr_is_possibly_under_txn(afr_transaction_type type, afr_local_t *local,
-                          xlator_t *this)
+                          unsigned int child_count)
```

Comment:
```suggestion
                          const unsigned int child_count)
```

(mainly for readability, the compiler probably already knows that)

Reply:
what's the benefit of declaring integer arguments as `const` ? the only thing that it will enforce if I'm not wrong is that the value of that variable can't be changed inside the function, but this variable is already a copy of the original one from the caller, so even if it's not declared as const and the code changes it, the original value in the caller is left untouched.